File: c/03-repeticao/main053.c
/*Faça um programa que leia a idade e o sexo de 5 pessoas, mostrando no final: 
a) Quantos homens foram cadastrados 
b) Quantas mulheres foram cadastradas 
c) A média de idade do grupo 
d) A média de idade dos homens 
e) Quantas mulheres tem mais de 20 anos 


Data 19/10/2025 -> 27ª Dia
*/

#include <stdio.h>
#include <string.h>


int main() {

    char sexo[2];
    int idade, i = 0, soma_grupo = 0,soma_homem = 0;
    int homem_cadastrado = 0;
    int mulheres_cadastrada = 0;
    int mulhere_mais_20 = 0;
    float media_grupo = 0,media_idade_homem = 0;

    for(i = 0; i < 5; i++){

        printf("Digite a sua idade: ");
        scanf("%d",&idade);

        printf("Digite o seu sexo: ");
        scanf("%s",sexo);

        soma_grupo += idade;

        if(strcmp(sexo,"m") == 0) {

            homem_cadastrado ++;
            soma_homem += idade;
        }
        else if(strcmp(sexo,"f") == 0) {

            mulheres_cadastrada ++;
            if(idade > 20) {

                mulhere_mais_20 ++;
            }

        }
    } 
    media_grupo = (float) soma_grupo / i; 
    media_idade_homem = (float) soma_homem / homem_cadastrado; 
    printf("a) Foi cadastrados no sistema %d Homens\n",homem_cadastrado);
    printf("b) Foi cadastrados no sistema %d Mulheres\n",mulheres_cadastrada);
    printf("c) A media de idade no grupo e: %.1f\n",media_grupo);
    printf("d) A media de idade dos Homens e: %.1f\n",media_idade_homem);
    printf("e) Tem %d muilheres com idade acima dos 20 anos",mulhere_mais_20);
    return 0;
}

File: python/03-repeticao/main053.py
'''Faça um programa que leia a idade e o sexo de 5 pessoas, mostrando no final: 
a) Quantos homens foram cadastrados 
b) Quantas mulheres foram cadastradas 
c) A média de idade do grupo 
d) A média de idade dos homens 
e) Quantas mulheres tem mais de 20 anos 


Data 19/10/2025 -> 27ª Dia
'''

cont = 0
homens_cadastrado = 0
mulheres_cadastrada = 0
media_grupo = 0
soma_grupo = 0
soma_homem = 0
media_idade_homem = 0
mulheres_mais_20 = 0

while cont < 5:

    idade = int(input('Digite a sua idade: '))
    sexo = str(input('Digite o seu sexo(m/f): ')).lower()

    soma_grupo += idade

    if sexo == 'm':
        homens_cadastrado +=1
        soma_homem += idade
    elif sexo == 'f':
        mulheres_cadastrada += 1
        if idade > 20:
            mulheres_mais_20 += 1
    
    cont += 1
media_grupo = soma_grupo / cont
media_idade_homem = soma_homem / homens_cadastrado
print(f'a) Foi cadastrados no sistema {homens_cadastrado} Homens')
print(f'b) Foi cadastradas no sistema {mulheres_cadastrada} Mulheres')
print(f'c) A media de idade do grupo é: {media_grupo}')
print(f'd) A media de idade dos Homens é: {media_idade_homem}')
print(f'e) Tem {mulheres_mais_20} Mulheres com a idade acima dos 20 anos')